### ISSUE TITLE
Docking to edge in an empty root split pane throws errors

### ISSUE THREAD

**Issue by ddaribo** [IgniteUI/igniteui-dockmanager#89]
## Description  
There is a dock manager with no panes initially, configured with the following layout: 
```
  this.dockManager.layout = {
            rootPane: {
                type: IgcDockManagerPaneType.splitPane,
                orientation: IgcSplitPaneOrientation.horizontal,
                panes: [],
                allowEmpty: true
            },
            floatingPanes: []
        };
```
The scenario involves adding a floating pane dynamically. On attempting to drag the pane and dock it to the edge of the root split pane, an error is thrown - view the gif in Attachments below.

 * version: any
 * browser: any

## Steps to reproduce  

1. Open [this sample](https://codesandbox.io/s/c-00233760-test-wc-xxjt2w?file=/src/index.ts)
2. Click the "Add floating pane" button
3. Try to dock the pane by choosing any of the up/down/left/right directions in the drop indicator when dragged over the empty root split pane.

## Result  
The following error is thrown:

```
TypeError
Cannot read properties of undefined (reading 'panes')
```
Call stack:
```
e.dockToEdge
e.dockPane
e.dragPaneEnd
e.handlePaneDragEnd
emitEvent
Object.emit
dragService.dragEnd
HTMLDocument.handlePointerUp
```

## Expected result  
The edge docking indicators should most probably not be shown in this case:
![MicrosoftTeams-image](https://github.com/IgniteUI/igniteui-dockmanager/assets/35433937/e97a9c4d-5212-4525-8bec-79faabd77ff0)

## Attachments  
![dm-dock-float-issue](https://github.com/IgniteUI/igniteui-dockmanager/assets/35433937/98e5e717-63ad-4873-bb7a-109286bd9c48)

**Comment by igdmdimitrov** [comment]
Fixed in v1.14.3

**Comment by tancyew** [comment]
hi just wanted to ask if v1.14.3 is packaged into the IgniteUI.Blazor.Trial as well? if yes what will be the new version number?

**Comment by ChronosSF** [comment]
Hey @tancyew ,

We just released Ignite UI for Blazor 23.2.20232.97 which contains the latest version of the Dock Manager.
